### PR TITLE
Parameterize setup

### LINKED ISSUE #4: Parameterize Setup via environment variables
Through [environment through Turborepo and dotenv](https://turbo.build/repo/docs/handbook/environment-variables) we could parameterize the setup. This will help in case someone already has a blocked port for e.g. `3000`, so smeargle couldn't load. Will also help in a later deployment i think.

### PR DESCRIPTION
**Pull request by robinduerhager**
Closes #4 

Internal Docker URLs like `http://drilbur` are untouched, since containernames are also hardcoded.
Though see #23 .

### CODE REVIEW

File: apps/smeargle/src/routes/index.tsx [hunk 1]
```diff
@@ -4,10 +4,15 @@ import { useRouteData } from 'solid-start';
...
+const {
+  VITE_RALTS_PORT
+} = import.meta.env
+
 async function fetchRalts() {
   console.log('server: ', isServer);
+  
   const response = await fetch(
-    isServer ? 'http://ralts:3005/' : 'http://localhost:3005/'
+    isServer ? `http://ralts:${VITE_RALTS_PORT || 3005}/` : `http://localhost:${VITE_RALTS_PORT || 3005}/`
```
Comment (ropfoo) [inline apps/smeargle/src/routes/index.tsx]: localhost should maybe be RALTS_DOMAIN for keeping things consistent

**Review by ropfoo — CHANGES_REQUESTED** [review]
Looks good but I think you forgot to change the domain in the ralts request [here](https://github.com/RopFoo/tonemato/pull/20/files#r1168178670)

**Comment by robinduerhager** [comment]
Fixed via [45be6ca](https://github.com/RopFoo/tonemato/pull/20/commits/45be6caaeebe99204b566888b605a8109f84d4fc)

**Review by ropfoo — COMMENTED** [review]
👍

**Review by ropfoo — COMMENTED** [review]
unfortunately i am to dumb to close this..

**Review by ropfoo — APPROVED** [review]

**Pull request merged**